CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+# Contributing to Numberscope
+
+## If you are new to software development, watch the video tutorials
+
+Watch the video series on contributing to Numberscope
+[here](https://www.youtube.com/playlist?list=PLA4KIQBQQRb5ccOdr9v0iLw_fKHup1PkU).
+This video series will introduce you to Numberscope, GitHub, the software we
+use in the Numberscope project, and the workflow for contributing to
+Numberscope.
+
```

Comment:
I haven't watched these as part of the review, I figured I'd start with the doc for now.  But it's great that you made them!  What an excellent resource to have!